g3/config.py

```diff
@@ -0,0 +1,36 @@
+import configparser
+from pathlib import Path
+from typing import Optional
+
+CONFIG_DIR = ".g3"
+CONFIG_FILE = "config.ini"
```

Comment:
Even if `.ini` extension helps about the config's format I would probably we skip it so that we are more similar to git's `~/.gitconfig`